polygeist/tools/cgeist/Lib/CGExpr.cc

```diff
@@ -1237,8 +1238,8 @@ ValueCategory MLIRScanner::VisitDeclRefExpr(DeclRefExpr *E) {
     // We need to decide where to put the Global.  If we are in a device
     // module, the global should be in the gpu module (which is nested inside
     // another main module).
-    std::pair<mlir::memref::GlobalOp, bool> gv = Glob.GetOrCreateGlobal(
-        VD, /*prefix=*/"", true,
+    std::pair<mlir::memref::GlobalOp, bool> gv = Glob.getOrCreateGlobal(
```

Comment:
Note: `GetORCreateGlobal` no longer that the parameter `tryInit`, if there is an initializer it initializes the global.